src/Text/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ElementPropTypes } from '@volusion/element-proptypes';
+import { ComponentProps } from '../../types';
+
+interface TextProps extends ComponentProps {
+    content: string;
+}
+
+const Text: React.FC<TextProps> = ({ content }) => {
```

Comment:
No `...rest` for passing other props?

Suggestion:
I'm preferable to the prop name `htmlString` since it tells you exactly what it is expecting.